modules/claims_api/lib/claims_api/claim_logger.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  # Logger for ClaimsApi, mostly used to format a print statement
+  # Uses the standard Rails logger
+  #
+  #   ClaimsApi::Logger.log('526ez', claim_id: 'id_here')
+  #   # => "ClaimsApi :: 526ez :: Claim ID: id_here"
```

Comment:
nitpick to go off of some of Jeff's earlier comments: making this example use `526` instead of `526ez`, since we avoid making that specific distinction elsewhere